src/adapters/scene.js

```diff
@@ -46,6 +47,8 @@ Scene.prototype.setupInitialPosition = async function(locationHash) {
     if (lastLocation) {
       this.center = [lastLocation.lng, lastLocation.lat];
       this.zoom = lastLocation.zoom;
+    } else if (window.initialBbox) {
```

Comment:
What do you think about rewriting this to a function returning directly an object to be merged with the Map options object., instead of storing stuff in `this` (which seem useless after initialization)? 
It would be called directly by `initMapBox`. It would make the whole init process easier to read and test.